stats/datadog/collector.go

```diff
@@ -69,6 +70,6 @@ func New(conf Config) (*common.Collector, error) {
 	return &common.Collector{
 		Config:      conf.Config,
 		Type:        "datadog",
-		ProcessTags: tagHandler(conf.TagBlacklist).processTags,
+		ProcessTags: tagHandler(conf.TagBlacklist.ToSystemTagSet()).processTags,
```

Comment:
Case in point, as far as I can see, this is the only usage of the `stats.TagSet..ToSystemTagSet()` method and it seems like a mistake. `tagHandler` seems like it should be `stats.TagSet`, not `stats.SystemTagSet`...

Reply:
https://github.com/loadimpact/k6/pull/1148#discussion_r334887048